ros2doctor/ros2doctor/api/package.py

```diff
@@ -110,21 +141,18 @@ def category(self):
     def check(self):
         """Check packages within the directory where command is called."""
         result = Result()
-        try:
-            distro_package_vers = get_distro_package_versions()
-            if not distro_package_vers:
-                result.add_error('ERROR: distro packages info is not found.')
-        except (AttributeError, RuntimeError, URLError):
```

Comment:
Are these exceptions never raised, or handled elsewhere?

Reply:
They are handled in `get_distro_package_versions()` https://github.com/ros2/ros2cli/pull/445/files#diff-08326eb5fdcea91201e23eeeada7edd2R31 due to this discussion https://github.com/ros2/ros2cli/pull/414#discussion_r371759216. To summarize, I move the error handling to `get_distro_package_versions` function to provide more information about exactly which part of `rosdistro` failed to be retrieved.